utilities/src/main/java/io/syndesis/qe/templates/SyndesisTemplate.java

```diff
@@ -181,22 +182,44 @@ private static void deployCrd() {
     }
 
     private static void deployOperator() {
-        log.info("Deploying operator from " + TestConfiguration.syndesisOperatorUrl());
-        final String output = OpenShiftUtils.binary().execute(
-            "create",
-            "-n", TestConfiguration.openShiftNamespace(),
-            "-f", TestConfiguration.syndesisOperatorUrl()
+
+        log.info("Generating resources using operator image " + TestConfiguration.syndesisOperatorUrl());
+
+        ProcessBuilder pb = new ProcessBuilder("docker",
+            "run",
+            "--entrypoint",
+            "operator",
+            TestConfiguration.syndesisOperatorUrl(),
```

Comment:
I guess I should create a new config prop rather than reuse this one

Reply:
:D maybe it is a good time to solve https://issues.jboss.org/browse/FUSEQE-3699